CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ Leaflet Changelog
 
 (all changes without author notice are by [@mourner](https://github.com/mourner))
 
+## 1.9.2 (2022-10-04)
+
+### 🐞 Bug fixes
+* Drop ESM entrypoint from package ([#8493](https://github.com/Leaflet/Leaflet/pull/8493) by [@jonkoops](https://github.com/jonkoops))
+* Check if `getElement()` is undefined before adding focus listener ([#8498](https://github.com/Leaflet/Leaflet/pull/8498) by [@Falke-Design](https://github.com/Falke-Design))
+
+### 🧪 Tests
+* Disable zoom animation for Line/PolyUtil tests ([#8478](https://github.com/Leaflet/Leaflet/pull/8478))
```

Comment:
Changelog should reflect the change in practice (what does it mean for others), not the underlying one:
```suggestion
* Fix unit tests suite stopping abruptly on Mac ([#8478](https://github.com/Leaflet/Leaflet/pull/8478))
```